code/modules/mob/living/carbon/human/species/shadekin/shadekin.dm

```diff
@@ -21,7 +21,7 @@
 	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws/shadekin, /datum/unarmed_attack/bite/sharp/shadekin)
 	rarity_value = 15	//INTERDIMENSIONAL FLUFFERS
 
-	siemens_coefficient = 0
+	siemens_coefficient = 1
```

Comment:
I vaguely remember talking to Soft about this in the past and I *think* it was intentional that they had a siemens coefficient of zero. That being said, I will just mark this and wait for their comment before requesting any change here.

Reply:
This was done originally to match the way that simplemobs were done at the time. (and because lorish wise, they would probably be able to absorb the energy) but it means that there's not really much that can be done to get them without lethals. 

So I'm fine with this change for making them reasonable to try to catch if they're careless.